frame/clearing-house/src/lib.rs

```diff
@@ -101,6 +115,10 @@ pub mod pallet {
 	/// Minimum margin ratio, below which liquidations can occur
 	type MaintenanceMarginRatio<T: Config> = StorageValue<_, T::Decimal, ValueQuery>;
 
+	#[pallet::storage]
+	/// Supported collateral asset ids
+	type CollateralTypes<T: Config> = StorageMap<_, Twox64Concat, AssetIdOf<T>, ()>;
+
```

Comment:
Btw, this 'hack' for having a set of allowable collateral types resulted from [this](https://substrate.stackexchange.com/q/685/587) discussion